_rules/html-page-lang-valid-bf051a.md

```diff
@@ -90,23 +90,23 @@ This `html` element has a `lang` attribute with a [known primary language tag][]
 
 #### Failed Example 1
 
-This `html` element has a `lang` attribute with a [known primary language tag][].
+This `html` element has a `lang` attribute, but it is not a [known primary language tag][].
```

Comment:
```suggestion
This `html` element has a `lang` attribute, with no [known primary language tag][].
```
The value of the `lang` attribute is not (and does not need to be) a primary language tag, only the, well, primary tag needs to be.